PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -67,7 +67,11 @@
 
 from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
 from PhysicsTools.PatAlgos.packedCandidateMuonID_cfi import packedCandidateMuonID
-(pp_on_AA_2018 | pp_on_PbPb_run3).toReplaceWith(slimmingTask, cms.Task(slimmingTask.copy(), packedCandidateMuonID))
+from PhysicsTools.PatAlgos.packedPFCandidateTrackChi2_cfi import packedPFCandidateTrackChi2
+lostTrackChi2 = packedPFCandidateTrackChi2.clone(candidates = "lostTracks", doLostTracks = True)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toReplaceWith(slimmingTask, cms.Task(slimmingTask.copy(), packedCandidateMuonID, packedPFCandidateTrackChi2, lostTrackChi2))
+
+
```

Comment:
```suggestion
(pp_on_AA_2018 | pp_on_PbPb_run3).toReplaceWith(slimmingTask, 
                                                cms.Task(slimmingTask.copy(), packedCandidateMuonID,
                                                         packedPFCandidateTrackChi2, lostTrackChi2))
```
not required, but something to make the update a bit more readable